software/go-proxy.yml

```
@@ -0,0 +1,10 @@
+name: "go-proxy"
+source_code_url: "https://github.com/yusing/go-proxy/tree/feat/godoxy-agent"
+website_url: "https://github.com/yusing/go-proxy/tree/feat/godoxy-agent"
+description: "An easy to use reverse proxy with docker integration. Also features idlesleeper, shutting down idle containers and starting them when traffic arrives."
+licenses:
+  - MIT
+platforms:
+  - Go
```

Deployment is also possible via docker.
```suggestion
platforms:
  - Docker
  - Go
```